src/Board.java

```diff
@@ -0,0 +1,100 @@
+import static java.lang.Character.isDigit;
+import java.util.*;
+/**
+ * The Board class models a typical board of Scrabble.
+ * The board is composed of 15 X 15 Squares. Each of these
+ * Squares contain a game Tile
+ *
+ * @author Michael Kyrollos, 101183521
+ * @version 1.0
+ * @date October 22, 2022
+ */
+
+
+import java.lang.*;
+public class Board {
+    public static final int SIZE = 15;
+    private Square squares[][];
+
+    public Board() {
+        squares = new Square[SIZE][SIZE];
+        for (int i = 0; i < SIZE; i++) {
+            for (int j = 0; j < SIZE; j++) {
+                squares[i][j] = new Square();
+            }
+        }
+    }
+
+    /**
+     *
+     * Returns True if word has been placed on board by player, false otherwise.
+     *
+     * @author Michael Kyrollos, ID: 101183521
+     *
+     * @return True if word has been placed, False otherwise.
+     *
+     */
+    public Boolean placeWord(Command command,ArrayList<Tile> tilesToPlay){
+        HashMap<Character,Integer> column = new HashMap<>();
```

Comment:
Make this a field of Board so it doesn't have to instantiate it every time placeWord() is called.